<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>821</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QFrame" name="headFrame">
       <property name="styleSheet">
        <string notr="true">#headFrame {
    background-color: rgb(1, 107, 13);
	border-radius: 10px
}
</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::Shape::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Shadow::Raised</enum>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item alignment="Qt::AlignmentFlag::AlignHCenter">
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <family>Cascadia Mono SemiBold</family>
            <pointsize>12</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Start Date : </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="dateEdit">
          <property name="font">
           <font>
            <family>Cascadia Code SemiBold</family>
            <pointsize>12</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="acceptDrops">
           <bool>false</bool>
          </property>
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignmentFlag::AlignHCenter">
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <family>Cascadia Code SemiBold</family>
            <pointsize>12</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>End Date :</string>
          </property>
          <property name="margin">
           <number>10</number>
          </property>
          <property name="indent">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="dateEdit_2">
          <property name="font">
           <font>
            <family>Cascadia Code SemiBold</family>
            <pointsize>12</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignmentFlag::AlignLeft">
         <widget class="QPushButton" name="pushButton_2">
          <property name="font">
           <font>
            <family>Cascadia Mono SemiBold</family>
            <pointsize>10</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">#headFrame QPushButton {
    background-color: blue;
    color: white;
    border: 2px solid darkblue;
    border-radius: 5px;
    padding: 5px 10px;
	margin-left: 30px;
}

#headFrame QPushButton:hover {
    background-color: lightblue;
    color: black;
    border-color: blue;
}

#headFrame QPushButton:pressed {
    background-color: darkblue;
    border-color: darkblue;
}
</string>
          </property>
          <property name="text">
           <string>Search In Range</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignmentFlag::AlignRight">
         <widget class="QPushButton" name="pushButton_3">
          <property name="font">
           <font>
            <family>Cascadia Mono SemiBold</family>
            <pointsize>10</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">#headFrame QPushButton {
    background-color: blue;
    color: white;
    border: 2px solid darkblue;
    border-radius: 5px;
    padding: 5px 10px;
}

#headFrame QPushButton:hover {
    background-color: lightblue;
    color: black;
    border-color: blue;
}

#headFrame QPushButton:pressed {
    background-color: darkblue;
    border-color: darkblue;
}
</string>
          </property>
          <property name="text">
           <string>Search All</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="tableWidget">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QTableWidget {
    background-color: #f0f0f0;
    border: 1px solid #dddddd;
}

QHeaderView::section {
    background-color: #4CAF50;
    color: white;
    padding: 4px;
    font-size: 15px;
    text-align: center;
	font: 700 15pt &quot;Arial&quot;;
}

QTableView QScrollBar:vertical {
    background: #f0f0f0;
    width: 10px;
    margin: 0px 0px 0px 0px;
}

QTableWidget::item {
    background-color: #ffffff;
    color: black;
}

QTableWidget::item:selected {
    background-color: #ccffcc;
}
</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::Shape::WinPanel</enum>
       </property>
       <property name="lineWidth">
        <number>2</number>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
       </property>
       <property name="textElideMode">
        <enum>Qt::TextElideMode::ElideLeft</enum>
       </property>
       <property name="gridStyle">
        <enum>Qt::PenStyle::SolidLine</enum>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
       <property name="rowCount">
        <number>4</number>
       </property>
       <property name="columnCount">
        <number>4</number>
       </property>
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <row/>
       <row/>
       <row/>
       <row/>
       <column/>
       <column/>
       <column/>
       <column/>
       <item row="0" column="0">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="0" column="3">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="1" column="0">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="1" column="1">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="1" column="2">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="2" column="1">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="3" column="0">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="3" column="2">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="3" column="3">
        <property name="text">
         <string/>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
